task/sast-snyk-check/0.4/sast-snyk-check.yaml

```diff
@@ -204,8 +204,9 @@ spec:
           snyk code test $ARGS --severity-threshold="$SEVERITY_THRESHOLD" "$resolved_path" --max-depth=1 --sarif-file-output="${resolved_path}/sast_snyk_check_out_${d//\//_}.json" 1>&2>> stdout.txt || true
         done
 
+
         # Merge all SARIF outputs
-        find "$SOURCE_CODE_DIR" -name "sast_snyk_check_out_*.json" -exec cat {} + > "${SOURCE_CODE_DIR}/sast_snyk_check_out.json"
+        find "$SOURCE_CODE_DIR" -name "sast_snyk_check_out_*.json" -exec jq -s '{"version": "2.1.0", "runs": map(.runs[])}' {} + >"${SOURCE_CODE_DIR}/sast_snyk_check_out.json" 2>/dev/null || echo '{"version": "2.1.0", "runs": []}' >"${SOURCE_CODE_DIR}/sast_snyk_check_out.json"
```

Comment:
As I said in my previous comment, this is not going to work because `csgrep` does not support multiple runs in a SARIF file: https://github.com/csutils/csdiff/blob/2791d449cede54a0e687d8c820da04bb4166d97b/src/lib/parser-json-sarif.cc#L149